apps/microtvm/pyproject.toml

```diff
@@ -98,22 +95,23 @@ onnxoptimizer = { version = "==0.2.6", optional = true }
 onnxruntime = { version = "==1.9.0", optional = true }
 
 # Pytorch (also used by ONNX)
-torch = { version = "==1.11.0", optional = true }
-torchvision = { version = "==0.12.0", optional = true }
+torch = { version = "==1.11.0" }
```

Comment:
how come you removed `optional = true` here? we use this so that you don't have to install these packages, but may opt to do so with e.g. `poetry install -E importer-torch`

Reply:
cc @mehrdadh who made this change

Reply:
this was a temporary hack to be able to export the dependencies in a requirement.txt file. We should revet this change and figure out how to generate dependencies file with extra dependencies.

Reply:
Looks like we no longer use requirement.txt file, so you can just revert changes in this file